core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -88,9 +87,10 @@ contract DesignatedVoting is Withdrawable {
 
     /**
      * @notice Forwards a reward retrieval to Voting.
-     * @dev rewards are added to the tokens already held by this contract.
+     * @dev Rewards are added to the tokens already held by this contract.
      * @param roundId defines the round from which voting rewards will be retrieved from.
      * @param toRetrieve an array of PendingRequests which rewards are retrieved from.
+     * @return totalRewardToIssue total amount of rewards returned to the voter.
```

Comment:
@chrismaree I addressed this NatSpec inconsistency in #1270

Reply:
great.